src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -498,7 +499,12 @@ def register_submission_to_objects_api(
             submission, object_mapping, REGISTRATION_ATTRIBUTE, object_data
         )
 
-        with get_objects_client() as objects_client:
+        objects_api_config = ObjectsAPIConfig.get_solo()
+        assert isinstance(objects_api_config, ObjectsAPIConfig)
```

Comment:
I assumed we use the default objects API group when registering to the Objects API from the ZGW registration.

Should we add support for group selection as well for ZGW?

Reply:
Yeah, selection seems best and consistent, but this can be a separate PR

Reply:
Opened https://github.com/open-formulieren/open-forms/issues/4344